consensus/ibft/pos_test.go

```diff
@@ -0,0 +1,146 @@
+package ibft
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
We use the `testify` package because it gives us a lot of flexibility that the regular `pkg.go.dev` couldn't provide.

If we were stuck using the regular golang package, we'd need to write all of our custom asserts - which `testify` provides for us since it is a wrapper over the standard go package.